### DISCUSSION TITLE
AWS Lambda/docker image pip install issue: cannot import name 'mean' from 'statistics'

### DISCUSSION THREAD

**Discussion by cyhsu** [geopandas/geopandas#2771] [Installation] [answered]
Hi All, 

I am trying to do `pip install geopandas>=0.12.1` in my docker images and push it to the AWS lambda. However, the AWS lambda show me the error message like below

```python
{
  "errorMessage": "Unable to import module 'macrozone_GeoJSON': cannot import name 'mean' from 'statistics' (/function/statistics.py)",
  "errorType": "Runtime.ImportModuleError",
  "requestId": "c9a5bf39-6280-43ac-b28d-5101e4490100",
  "stackTrace": []
}
```

I tried to downgrade the version to 0.11, adding scipy version, and other geopandas dependencies;  but it is still the same. Does anyone know how to resolve this?

Thank you.

The following script is my requirements.txt
```
numpy>=1.22.4
pandas>=1.4.3
scipy>=1.8.1
PyYAML~=5.4.1
scipy>=1.8.1
h3>=3.7.4
Jinja2>=3.1.2
netCDF4>=1.5.8
rasterio>=1.2.10
xarray>= 2022.3.0
dask>=2022.05.1
distributed>=2022.7
rioxarray~=0.11.1
s3fs~=2022.5.0
pyproj>=3.3.1
google-cloud-bigquery>=3.1.0
google-cloud-bigquery-storage>=2.13.1
google-api-python-client>=2.49.0
Shapely>=1.8.2
db-dtypes>=1.0.1
hvac>=0.11.2
scikit-learn>=1.2.1
geopandas>=0.11
fiona>=1.8
rtree
```

**Comment by martinfleis** [comment, marked as answer]
Do you have a file called "statistics" somewhere in the path that Python may prioritise over the `statistics` built-in package? 

Btw, I am nearly 100% sure this issue has nothing to do with geopandas apart from it being part of the environment.
